pypeit/core/arc.py

```diff
@@ -478,9 +480,11 @@ def detect_lines(censpec, nfitpix=5, sigdetect = 10.0, FWHM = 10.0, cont_samp =
       The variance on tcent
     w : ndarray
       An index array indicating which detections are the most reliable.
-    detns : ndarray
-      The spectrum used to find detections. This spectrum has
-      had any "continuum" emission subtracted off
+    arc : ndarray
+      The continuum sutracted arc used to find detections.
+    nsig : ndarray
+      The significance of each line detected relative to the 1sigma variation in the continuum subtracted arc in the
+      the line free region. Bad lines are assigned a significance of -1, since they don't have an amplitude fit
```

Comment:
add `nsig` to the Return doc string and define it.

Reply:
I'm not following you. The code above is exactly the documentation for nsig in the return doc string.